modules/metastorage/src/integrationTest/java/org/apache/ignite/internal/metastorage/impl/ItMetaStorageMultipleNodesAbstractTest.java

```diff
@@ -228,6 +233,8 @@ void testLearnerJoin(TestInfo testInfo) throws NodeStoppingException {
 
         firstNode.cmgManager.initCluster(List.of(firstNode.name()), List.of(firstNode.name()), "test");
 
+        firstNode.deployWatches();
```

Comment:
It seems incorrect. deployWatches is a part of node start procedure.

Reply:
Fixed